packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -392,49 +388,41 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
             exchangeRateInCents,
           );
 
-          const lastRemainingHbars = remainingHbarsBefore;
-          let spentToday = 0;
-
           //Unlinking the ipAdress, since ipAddress when running tests in CI and locally is the same
-          const basicPlans = await hbarSpendingPlanRepository.findAllActiveBySubscriptionType(
-            SubscriptionType.BASIC,
+          const basicPlans = await hbarSpendingPlanRepository.findAllActiveBySubscriptionTier(
+            [SubscriptionTier.BASIC],
             requestDetails,
           );
           const ipPlans = await ipSpendingPlanRepository.getAllPlans(requestDetails);
           const ipPlanWithId = await basicPlans.map((plan) => ipPlans.find((ipPlan) => ipPlan.planId === plan.id));
-          const ipAddress = ipPlanWithId[0].ipAddress;
-          await ipSpendingPlanRepository.delete(ipAddress, requestDetails);
+          const ipAddress = ipPlanWithId[0]?.ipAddress;
+          if (ipAddress) {
+            await ipSpendingPlanRepository.delete(ipAddress, requestDetails);
+          }
           expect(ethAddressSpendingPlanRepository.findByAddress(accounts[2].address, requestDetails)).to.be.rejected;
-          expect(remainingHbarsBefore).to.be.gt(0);
-          let spendingPlanId;
           try {
             for (let i = 0; i < 50; i++) {
               await deployContract(largeContractJson, accounts[2].wallet);
-              const remainingHbars = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
-
-              // Check that the BASIC spending plan is created after the first deployment only
-              if (i === 0) {
```

Comment:
@quiet-node Why is this deleted? I wanted to have this check here to ensure we create a basic plan as intended?